apps/website/src/components/LeftSidebarContent.astro

```diff
@@ -169,6 +170,19 @@ sections.set(
       color: var(--color-text);
       position: relative;
 
+      // scroll to current link when page loads, without any JS.
+      // based on https://oh-snap.netlify.app/#scroll-start-trick
```

Comment:
the JS approach would just query for `[aria-current=page]` and call `scrollIntoView` on it, no need for nanostores.

and yes, nanostores keeps the state in memory so it would be lost anyway